src/pages/SettingsPage.js

```diff
@@ -81,11 +81,11 @@ const Settings = ({
 
   const onCheckConnection = () => {
     //  Hash the password.
-    const authURL = `${syncURL}${AUTH_ENDPOINT}`;
+
     const syncSiteName = UIDatabase.getSetting(SETTINGS_KEYS.SYNC_SITE_NAME);
     const syncAuthenticator = new SyncAuthenticator(AppSettings);
     syncAuthenticator
-      .authenticate(authURL, syncSiteName, null, currentUserPasswordHash)
+      .authenticate(syncURL, syncSiteName, null, currentUserPasswordHash)
```

Comment:
Sorry, we also need to change this password.. should be `UIDatabase.getSetting(SETTING_KEY.SYNC_SITE_PASSWORD_HASH)`

Reply:
No worries.
This ae94f46 should make you happy.